### ISSUE TITLE
2.2.3 Regression: babel-node required in non-dev-dependencies

### ISSUE THREAD

**Issue by LeoniePhiline** [tailwindlabs/tailwindcss#4784]
### What version of Tailwind CSS are you using?

2.2.3

### What build tool (or framework if it abstracts the build tool) are you using?

vite 2

### What version of Node.js are you using?

14

### What browser are you using?

N/A

### What operating system are you using?

Linux

### Reproduction repository

https://github.com/tailwindlabs/tailwindcss/commit/c0ee52060ca237d0b3a617b99983b8d747ce8e6b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

### Describe your issue

https://github.com/tailwindlabs/tailwindcss/blob/master/package.json#L29-L34

npm "postinstall" calls `"generate:plugin-list": "babel-node scripts/create-plugin-list.js"`.

But `@babel/node` is in `devDependencies` and thus not installed in projects depending on tailwindcss. (https://github.com/tailwindlabs/tailwindcss/blob/master/package.json#L50)

This makes update and install fail:

```
❯ npm update
npm ERR! code 127
npm ERR! path /foo/bar/project/node_modules/tailwindcss
npm ERR! command failed
npm ERR! command sh -c npm run generate:plugin-list
npm ERR! > tailwindcss@2.2.3 generate:plugin-list
npm ERR! > babel-node scripts/create-plugin-list.js
npm ERR! sh: Line 1: babel-node: Command not found.
```

**Comment by batnieluyo** [comment]
In 2.1.2 version :

```
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! code 127
npm ERR! path /Users/Me/Sites/Tests/node_modules/tailwindcss
npm ERR! command failed
npm ERR! command sh -c npm run generate:plugin-list
npm ERR! > tailwindcss@2.2.3 generate:plugin-list
npm ERR! > babel-node scripts/create-plugin-list.js
npm ERR! sh: babel-node: command not found

```

**Comment by LeoniePhiline** [comment]
@batnieluyo That must be while you `npm install` or `npm update`? 

The change was only published with 2.2.3.
See https://github.com/tailwindlabs/tailwindcss/commit/c0ee52060ca237d0b3a617b99983b8d747ce8e6b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

Wait for 2.2.4, it should be there very soon. @sachinraja has already created a quick fix PR https://github.com/tailwindlabs/tailwindcss/pull/4783

**Comment by jonnitto** [comment]
Thanks to @sachinraja for providing such a quick fix! ❤️

**Comment by adamwathan** [comment]
Should be fixed in v2.2.4, sorry about that.
